src/cryptenroll/cryptenroll.c

```diff
@@ -240,6 +249,16 @@ static int parse_argv(int argc, char *argv[]) {
                         break;
                 }
 
+                case ARG_FIDO2_CRED_ALG: {
+                        if (isempty(optarg))
+                                break;
+
+                        r = parse_fido2_algorithm(optarg, &arg_fido2_cred_alg);
+                        if (r < 0)
+                                return log_error_errno(SYNTHETIC_ERRNO(r), "Failed to parse COSE algorithm: %s", optarg);
```

Comment:
Please drop SYNTHETIC_ERRNO(), which is used when an errno is explicitly specified, e.g. `SYNTHETIC_ERRNO(EINVAL)`.
So,
```
return log_error_errno(r, "Failed to ...", ...);
```
is enough.

Reply:
Thanks for your review. Done.